File: data/java/BINARY_REPRESENTATION_OF_NEXT_NUMBER_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<String> param0 = new ArrayList<>();
    param0.add("DXh");
    param0.add("48703586411816");
    param0.add("0001");
    param0.add("yWg WvjNKS");
    param0.add("8408568459");
    param0.add("01");
    param0.add("DFECZ CWtN");
    param0.add("37742236");
    param0.add("001101");
    param0.add("LDxERLmYn");
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result\n");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
            builder.append("BINARY_REPRESENTATION_OF_NEXT_NUMBER," + args[1] + "," + i + "," + f_gold(param0.get(i)) + "\n");
        } catch (Exception e) {
            builder.append("BINARY_REPRESENTATION_OF_NEXT_NUMBER," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}

File: data/python/BINARY_REPRESENTATION_OF_NEXT_NUMBER_MAIN.py
if __name__ == '__main__':
    param = [
    ('DXh',),
    ('48703586411816',),
    ('0001',),
    ('yWg WvjNKS',),
    ('8408568459',),
    ('01',),
    ('DFECZ CWtN',),
    ('37742236',),
    ('001101',),
    ('LDxERLmYn',)
        ]
    n_success = 0
    for i, parameters_set in enumerate(param):
        try:
            if f_filled(*parameters_set) == f_gold(*parameters_set):
                print("BINARY_REPRESENTATION_OF_NEXT_NUMBER," + sys.argv[1] + "," + str(i) + ",SUCCESS")
            else:
                print("BINARY_REPRESENTATION_OF_NEXT_NUMBER," + sys.argv[1] + "," + str(i) + ",FAILURE")
        except:
            print("BINARY_REPRESENTATION_OF_NEXT_NUMBER," + sys.argv[1] + "," + str(i) + ",EXCEPTION")
